packages/cli/src/commands/dev-contracts.ts

```diff
@@ -57,6 +59,10 @@ const commandModule: CommandModule<Options, Options> = {
 
     // Start an anvil instance in the background if no RPC url is provided
     if (!args.rpc) {
+      console.log(chalk.gray("Cleaning devnode cache"));
+      const userHomeDir = homedir();
+      rmSync(path.join(userHomeDir, ".foundry", "anvil", "tmp"), { recursive: true, force: true });
```

Comment:
this corresponds to anvil's cache location as configured here: https://github.com/foundry-rs/foundry/blob/c331b5eeee1b4151ef7354a081667e2d770b37f5/anvil/src/config.rs#L1085-L1086

Reply:
opened an issue on anvil's side here: https://github.com/foundry-rs/foundry/issues/4989